.github/workflows/black.yml

```diff
@@ -0,0 +1,22 @@
+name: black
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install Black
+      run: |
+        python -m pip install --upgrade pip
+        pip install Black
+    - name: run black
+      run: |
+        black . --check
```

Comment:
Instead of a GitHub action, this should replace the flake8-based `lint` job in the tox config.

Reply:
Or better yet, we should use flake8-black and remove any redundant plugins.

Reply:
Hi @rpkilby @carltongibson or would you prefer to move all lint jobs to GitHub actions? 

Re flake8-black, I've not used it before. I'm interested in your views on the pro/cons the benefit of using a small add-in (flake8-black) compared to using the check option of the tool itself. 

This project validates isort via the isort--check-only option. 
https://github.com/carltongibson/django-filter/blob/4c78f08667bc8dfe4af9417c38c4e62dc61bff7f/tox.ini#L28